env_installer/construct.yaml

```diff
@@ -21,6 +21,7 @@ license_file: ../LICENSE
 specs:
   - conda
   - ipywidgets 7.6.5
+  - ipympl 0.8.2
```

Comment:
+1 seems reasonable to me.
Do we want/need the version pin?

Reply:
Nope! It should be at least `>=0.8.2` but the spaces confused me because it's different than the syntax a normal conda env file?

Reply:
https://github.com/conda/constructor/blob/master/CONSTRUCT.md#specs says that normal conda env sntax should work. So I guess `ipympl >=0.8.2` should do it.

Reply:
does conda support version statements like `ipympl ~=0.8.2` ?

https://www.python.org/dev/peps/pep-0440/#compatible-release

Reply:
I can say pretty confidently that for ipympl you probably want `>=`. The version numbers there pretty much only reflect improvements to the widget and don't reflect compatibility with jupyterlab

Reply:
but good thought! and probably relevant in the future